### PR TITLE
Upgrade cozy-client-js :rocket:

### PR DESCRIPTION
**Pull request by kosssi**

### CODE REVIEW

File: mobile/src/actions/settings.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ export const registerDevice = () => async (dispatch, getState) => {
   await cozy.client.authorize().then(async ({ client }) => {
     dispatch(setClient(client))
     await cozy.client.offline.replicateFromCozy('io.cozy.files')
+    cozy.client.offline.startRepeatedReplication('io.cozy.files', 15)
```
Comment (enguerran) [inline mobile/src/actions/settings.js]: Do we have to `replicateFromCozy` and `startRepeatedReplication` sequencially. Don't we have to `replicateFromCozy.then(startRepeatedReplication)`?

File: mobile/src/main.jsx [hunk 2]
```diff
@@ -66,6 +66,7 @@ document.addEventListener('DOMContentLoaded', () => {
         })
       } else {
         initClient(url)
+        cozy.client.offline.startRepeatedReplication('io.cozy.files', 15)
```
Comment (enguerran) [inline mobile/src/main.jsx]: If we set up the `replication` workflow in a way (cf. https://github.com/cozy/cozy-files-v3/pull/142/files#r104361067), don't we have to start it here the same way: first `replicateFromCozy` then `startRepeatedReplication`?

File: mobile/src/actions/settings.js (hunk 1 above)
Comment (kosssi) [inline mobile/src/actions/settings.js]: `replicateFromCozy have an await, so the replication is finish and we launch the repeated replication.
When I remove `replicationFromCozy` this app failed (OPEN_FOLDER_FAILURE) on first connection.

File: mobile/src/main.jsx (hunk 2 above)
Comment (kosssi) [inline mobile/src/main.jsx]: the first replication is only here to initialize `io.cozy.files.root-dir` ;)

**Comment by enguerran** [comment]
Now that it is clearer, I approve and merge this PR.

**Comment by kosssi** [comment]
👍

**Pull request merged**